scripts/ceph-windows/run_tests

```diff
@@ -46,16 +49,29 @@ function collect_artifacts() {
     rm -rf $WORKSPACE/artifacts
     mkdir -p $WORKSPACE/artifacts
 
+    SSH_USER=$UBUNTU_SSH_USER SSH_ADDRESS=$UBUNTU_VM_IP ssh_exec $WORKSPACE/ceph/build/bin/ceph status
+
     scp_download /workspace/test_results $WORKSPACE/artifacts/test_results
     if [[ "$INCLUDE_USERSPACE_CRASH_DUMPS" = true ]]; then
         scp_download /userspace_crash_dumps $WORKSPACE/artifacts/userspace_crash_dumps
     fi
     if [[ "$INCLUDE_CEPH_ZIP" = true ]]; then
         cp $WORKSPACE/ceph.zip $WORKSPACE/artifacts/ceph.zip
     fi
+
+    mkdir -p $WORKSPACE/artifacts/client
+
+    scp_download /ProgramData/ceph/logs $WORKSPACE/artifacts/client/logs
+    cp $CEPH_WINDOWS_CONF $WORKSPACE/artifacts/client
+    ssh_exec /wnbd/wnbd-client.exe version
+    SCRIPT_DIR=$(dirname "$BASH_SOURCE") scp_upload "$SCRIPT_DIR/collect-event-logs.ps1" $WORKSPACE
```

Comment:
The SCP destionation location is `$WORKSPACE`, and it seems wrong, as this is local path on the KVM host with the Windows VM.

Judging by the next script lines, the destination should be `/workspace`.